examples/decoding/plot_ssd_spatial_filters.py

```diff
@@ -0,0 +1,142 @@
+"""
+===========================================================
+Compute Sepctro-Spatial Decomposition (SDD) spatial filters
+===========================================================
+In this example, we will compute spatial filters for retaining
+oscillatory brain activity and down-weighting 1/f background signals
+as proposed by :footcite:`NikulinEtAl2011`.
+The idea is to learn spatial filters that separate oscillatory dynamics
+from surrounding non-oscillatory noise based on the covariance in the
+frequency band of interest and the noise covariance absed on surrounding
+frequencies.
+References
```

Comment:
```suggestion

References
```

Reply:
I would also move this section to the very bottom in its own `##########`-separated section